app/controllers/patients_controller.rb

```diff
@@ -75,6 +65,9 @@ def patient_params
       :household_size_adults, :household_size_children, :insurance,
       :referred_by, :initial_call_date, :urgent_flag,
       :clinic_id,
+
+      :last_menstrual_period_days, :last_menstrual_period_weeks,
```

Comment:
Yeah, they're redundant, and straightening out strong params definitely slipped my mind, so thanks for flagging this! Since there's no longer a `pregnancy` object here, we can move them all over and totally eliminate the nesting. I might do a little bit of additional cleanup refactoring here by moving sets of attributes into constants, reflecting where they are in the form (e.g. `PATIENT_DASHBOARD_PARAMS = [:name, :primary_phone, :appointment_date, ...]`).

(In the unlikely event that you haven't seen these before -- Strong parameters are one of rails' most 'Why the isn't this working' pieces of the framework, but they're great. They make it so you can't mass assign attributes thru a form unless all the attributes are cleared. The API writeup isn't bad: http://edgeguides.rubyonrails.org/action_controller_overview.html#strong-parameters )